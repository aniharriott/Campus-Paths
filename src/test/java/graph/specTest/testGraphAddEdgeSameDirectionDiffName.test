# Test Graph addEdge same direction different name file.
# Tests if two edges can be added that have the
# same source and destination, but different names.

# Create Graph
CreateGraph graph1

# Add nodes
AddNode graph1 n1
AddNode graph1 n2

# Add edges
AddEdge graph1 n1 n2 e1
AddEdge graph1 n1 n2 e2