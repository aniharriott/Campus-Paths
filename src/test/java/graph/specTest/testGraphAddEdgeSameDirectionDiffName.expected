# Test Graph addEdge same direction different name file.
# Tests if two edges can be added that have the
# same source and destination, but different names.

# Create Graph
created graph graph1

# Add nodes
added node n1 to graph1
added node n2 to graph1

# Add edges
added edge e1 from n1 to n2 in graph1
added edge e2 from n1 to n2 in graph1